netket/operator/_sumoperators.py

```diff
@@ -0,0 +1,43 @@
+from typing import Optional, List
+
+from netket.utils.types import DType
+
+from netket.hilbert import AbstractHilbert
+from netket.operator import ContinousOperator
+
+import jax.numpy as jnp
+
+
+class SumOperator(ContinousOperator):
+    def __init__(
+        self,
+        hilbert: AbstractHilbert,
+        operators: List,
+        coeff: float,
+        dtype: Optional[DType] = float,
+    ):
+        r"""Args:
+        operators: A list of ContinousOperator objects
+        coeff: A coefficient for each ContinousOperator object
+        """
+        self._ops = operators
+        self._coeff = coeff
+        self._dtype = dtype
+
+        super().__init__(hilbert)
+
+    @property
+    def dtype(self) -> DType:
+        return self._dtype
+
+    def expect_kernel(self, logpsi, params, x, data):
+        result = [
+            op.expect_kernel(logpsi, params, x, data[i])
+            for i, op in enumerate(self._ops)
+        ]
+        return jnp.sum(jnp.array(result))
```

Comment:
this should be 
```suggestion
        return jnp.sum(jnp.vstack(result), axis=0)
```
so that the local terms are added together and then we can properly estimate the variance